app/controllers/messages_controller.rb

```diff
@@ -6,10 +6,6 @@ class MessagesController < ApplicationController
     controller.ensure_logged_in t("layouts.notifications.you_must_log_in_to_send_a_message")
   end
 
-  before_filter do |controller|
-    controller.ensure_authorized t("layouts.notifications.you_are_not_authorized_to_do_this")
-  end
-
```

Comment:
Ok, seems that in MessagesController the same error message is shown but in PeopleController it's then not. Instead the actions are just silently ignored. Do I interpret this correctly? Is this deliberate?